lib/puppet/provider/user/useradd.rb

```diff
@@ -7,7 +7,10 @@
 Puppet::Type.type(:user).provide :useradd, :parent => Puppet::Provider::NameService::ObjectAdd do
   desc "User management via `useradd` and its ilk.  Note that you will need to
     install Ruby's shadow password library (often known as `ruby-libshadow`)
-    if you wish to manage user passwords."
+    if you wish to manage user passwords.
+
+    To use `forcelocal` parameter, you need to install the `libuser` package (providing
```

Comment:
to match the other docstring
```suggestion
    To use the `forcelocal` parameter, you need to install the `libuser` package (providing
```